sljit_src/sljitNativeARM_32.c

```diff
@@ -1502,8 +1507,16 @@ static SLJIT_INLINE sljit_s32 emit_single_op(struct sljit_compiler *compiler, sl
 	case SLJIT_CTZ:
 		SLJIT_ASSERT(!(flags & INV_IMM) && !(src2 & SRC2_IMM));
 		SLJIT_ASSERT(src1 == TMP_REG1 && !(flags & ARGS_SWAPPED));
+#if defined(SLJIT_CONFIG_ARM_V7) && SLJIT_CONFIG_ARM_V7
 		FAIL_IF(push_inst(compiler, RBIT | RD(dst) | RM(src2)));
 		return push_inst(compiler, CLZ | RD(dst) | RM(dst));
+#else
+		FAIL_IF(push_inst(compiler, RSBS | RD(dst) | RN(src2)));
```

Comment:
Is this RSB? Setting flags are not part of the instruction, should be added after a `|`

Reply:
it is a second encoding[1] for `RSB` which uses an immediate as one of the parameters and that was previously unused.  I enabled the "reporting" bit by default but I see why you would rather have that set explicitly at invocation time, and probably only to the one invocation that matters, eventhough it probably doesn't make any difference performance/logic wise in this context.

would it better to use `RSBI` as a mnemonic?

[1] https://developer.arm.com/documentation/ddi0597/2023-03/Base-Instructions/RSB--RSBS--immediate---Reverse-Subtract--immediate--

Reply:
except it was being used through `SRC2_IMM` but I wasn't familiar enough with the code, eitherway code has been discarded for now